src/mon/PGMap.cc

```diff
@@ -2886,7 +2886,7 @@ int reweight::by_utilization(
     for (const auto& pg : pgm.pg_stat) {
       if (pools && pools->count(pg.first.pool()) == 0)
 	continue;
-      for (const auto acting : pg.second.acting) {
+      for (const auto acting : pg.second.up) {
```

Comment:
this change is for "reweight by pg". mostly amount of used will be in accordance with pg number of this osd.